docs/spark/tutorials/hdinsight-deployment.md

```diff
@@ -7,126 +7,176 @@ ms.custom: mvc
+* If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free/).
+* Sign in to the [Azure portal](https://portal.azure.com/).
+* Install Azure Storage Explorer on your [Windows](https://go.microsoft.com/fwlink/?LinkId=708343&clcid=0x409), [Linux](https://go.microsoft.com/fwlink/?LinkId=722418&clcid=0x409), or [MacOS](https://go.microsoft.com/fwlink/?LinkId=708342&clcid=0x409) computer.
+* Complete the [.NET for Apache Spark - Get Started in 10-Minutes](https://dotnet.microsoft.com/learn/data/spark-tutorial/intro) tutorial.
 
-## Prepare worker dependencies
+## Access your storage accounts
 
-**Microsoft.Spark.Worker** is a backend component that lives on the individual worker nodes of your Spark cluster. When you want to execute a C# UDF (user-defined function), Spark needs to understand how to launch the .NET CLR to execute the UDF. **Microsoft.Spark.Worker** provides a collection of classes to Spark that enable this functionality.
+1. Open Azure Storage Explorer.
 
-1. Select a [Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases) Linux netcoreapp release to be deployed on your cluster.
+2. Select **Add Account** on the left menu, and sign in to your Azure account.
 
-   For example, if you want `.NET for Apache Spark v0.1.0` using `netcoreapp2.1`, you'd download [Microsoft.Spark.Worker.netcoreapp2.1.linux-x64-0.1.0.tar.gz](https://github.com/dotnet/spark/releases/download/v0.1.0/Microsoft.Spark.Worker.netcoreapp2.1.linux-x64-0.1.0.tar.gz).
+    ![Sign in to Azure account from Storage Explorer](./media/hdinsight-deployment/signin-azure-storage-explorer.png)
 
-2. Upload `Microsoft.Spark.Worker.<release>.tar.gz` and [install-worker.sh](https://github.com/dotnet/spark/blob/master/deployment/install-worker.sh) to a distributed file system (e.g., HDFS, WASB, ADLS) that your cluster has access to.
+   After you sign in, you should see all storage accounts you have and any resources you have uploaded to your storage accounts.
 
-## Prepare your .NET for Apache Spark app
+## Create an HDInsight cluster
 
-1. Follow the [Get Started](get-started.md) tutorial to build your app.
+> [!IMPORTANT]  
+> Billing for HDInsight clusters is prorated per minute, even if you're not using them. Be sure to delete your cluster after you have finished using it. For more information, see the [Clean up resources](#clean-up-resources) section of this tutorial.
 
-2. Publish your Spark .NET app as self-contained.
+1. Visit the [Azure portal](https://portal.azure.com).
 
-   You can run the following command on Linux.
+2. Select **+ Create a resource**. Then, select **HDInsight** from the **Analytics** category.
 
-   ```dotnetcli
-   dotnet publish -c Release -f netcoreapp2.1 -r ubuntu.16.04-x64
+    ![Create HDInsight resource from Azure portal](./media/hdinsight-deployment/create-hdinsight-resource.png)
```

Comment:
This is showing up as creating a new Azure Databricks workspace picture, not the HDI picture.